component/module/git/internal/vcs/auth/auth.go

```diff
@@ -0,0 +1,67 @@
+package auth
```

Comment:
I don't think it's necessary to create a dedicated package for auth; you can just put this file in the upper vcs package.

Reply:
The second after I pushed the PR I thought the same thing. Updated it so it's under the vcs package.